db/primary_migrate/20240801155943_change_in_person_enrollment_sponsor_id_to_non_nullable.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeInPersonEnrollmentSponsorIdToNonNullable < ActiveRecord::Migration[7.1]
+  def change
+    add_check_constraint :in_person_enrollments, "sponsor_id IS NOT NULL", name: "in_person_enrollments_sponsor_id_null", validate: false
+  end
+end
+
+class ValidateChangeInPersonEnrollmentSponserIdToNonNullable < ActiveRecord::Migration[7.1]
+  def up
```

Comment:
two separate migration classes should probably be in two separate files, makes it clearer how to handle/recover if one fails

Reply:
Great call!